src/core/Header/HeaderBreadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+
+import React from 'react';
+
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/header.css';
+
+export type HeaderBreadcrumbsTranslations = {
+  /**
+   * `nav` element aria-label.
+   */
+  breadcrumbs: string;
+};
+
+export type HeaderBreadcrumbsProps = {
+  /**
+   * Array of elements, chevrons will be put between them.
+   * (expects HeaderButton[])
+   * @example
+   * [
+   *  <HeaderButton key={...} ... />
+   *  <HeaderButton key={...} ... />
+   * ]
+   */
+  items: React.ReactNode[];
+  /**
+   * Provide localized strings.
+   */
+  translatedStrings?: HeaderBreadcrumbsTranslations;
+};
+
+const defaultTranslations: HeaderBreadcrumbsTranslations = {
+  breadcrumbs: 'breadcrumbs',
+};
+
+/**
+ * Header breadcrumbs, adds chevron between each provided items.
+ * @example
+ * <HeaderBreadcrumbs
+ *   items={[
+ *     <HeaderButton key={...} ... />
+ *     <HeaderButton key={...} ... />
+ *   ]}
+ * />
+ */
+export const HeaderBreadcrumbs = (props: HeaderBreadcrumbsProps) => {
+  const { items, translatedStrings } = props;
+  const breadcrumbsTranslations = {
+    ...defaultTranslations,
+    ...translatedStrings,
+  };
+
+  useTheme();
+  return (
+    <nav aria-label={breadcrumbsTranslations.breadcrumbs}>
```

Comment:
If you pass `...rest` here, I think this won't need translations, right?

Reply:
Yes, but it doesnt really makes it obvious that we are passing in english values in there that should be translated. What would be the type of the `HeaderBreadcrumbsProps` ?

Reply:
I don't think it needs a specific type. Just passing rest is enough. All JSX components can accept `data-` and `aria-` attributes without a special type.

As for the "makes it obvious" debate, you can say the same thing for the prop itself. It's not obvious that the translation is used for aria-label.

Reply:
I dont want to debate I just want to know what to put in there according to iTwinUI standards, so we can finalize this...

Reply:
Changed to use `{...rest}` on the `nav` component.